### ISSUE TITLE
8.8 Release

### ISSUE THREAD

**Issue by mattrobenolt** [getsentry/sentry#4066]
As per last month (GH-3844) plz supply `CHANGES` here that will get tagged and pushed out today.

Thanks ❤

@getsentry/team

**Comment by benvinegar** [comment]
For new contributors, here's how I mostly figure this out:

* Look up [your commits](https://github.com/getsentry/sentry/commits/master?author=benvinegar) in the past month (change the `author` query string in the URL)
* Identify anything major enough to be communicated to OS Sentry users (e.g. new features, major behavior changes, etc)
* Add them to this thread

---

My additions:

* _Literally nothing_

**Comment by mattrobenolt** [comment]
Full shortlog:

```
Armin Ronacher (25):
      Workaround for a concurrency bug with date parsing
      Do not fail if there are no stacks in threads
      Moved strptime code
      Merge pull request #3830 from getsentry/bugfix/empty-stacks
      Merge pull request #3788 from getsentry/bugfix/date-concurrency
      warning -> error for frame symbolication
      Merge pull request #3830 from getsentry/bugfix/empty-stacks
      Merge pull request #3788 from getsentry/bugfix/date-concurrency
      Do not fail if symsynd is not available for tests
      Fix vmaddr handling and disable error logging for symbolication due to spam.
      Merge pull request #3892 from getsentry/bugfix/vmaddr
      Merge pull request #3882 from getsentry/bugfix/missing-symsynd
      Fix bad address padding code for ios
      Added a test for address padding
      Merge pull request #3907 from getsentry/bugfix/bad-pad
      Show filename in a tooltip if otherwise not shown in UI.  Fixes #3987
      Added more lenient stack handling to improve symbolication
      Merge pull request #4010 from getsentry/feature/lenient-stack
      Fixed a bad logic bug in the prioritization of module names in the frame
      Merge pull request #3998 from getsentry/feature/cs-filenames
      Added support mail config flag
      Update u2f failure dialog to show support
      Add FLAG_PRIORITIZE_DISK to system.support-email
      Added absolute import
      Merge pull request #4043 from getsentry/feature/support-mail-in-u2f

Ben Vinegar (10):
      Bump raven-js to 3.4.0 (#3890)
      Bump raven-js to 3.4.1 (#3917)
      Bump raven-js to 3.5.0 (#3957)
      Add note about how to update integration docs for /settings/install (#3973)
      Bump raven-js dependency to test getsentry/raven-js#582 (#4004)
      Pre-fill registration form email w/ invite email (#4032)
      Bump raven-js to 3.6.0
      Bump raven-js to 3.6.0 (#4049)
      Cleaner ui for assigning members to teams (#4055)
      Bump raven-js to 3.6.1 (#4063)

Chris Jennings (5):
      Bigger pinned tab icon (#3867)
      Bigger pinned tab icon (#3867)
      Invert report arrows (#3925)
      Merge pull request #3946 from getsentry/spacing_utilities
      Merge pull request #3960 from getsentry/reduce-importants

David Chavez (1):
      Fix grammar in empty query results message (#3879)

David Cramer (108):
      Vendor django-social-auth
      Handle global dates with missing env/release
      Merge pull request #3858 from getsentry/global-seen-stats
      Vendor django-social-auth
      Various fixes for linter checks (not registered)
      Remove abspath
      We require Python requirements for linting JS
      Overhaul internal admin
      Add generate_reset_password_link
      Basic test for md5_from_hash
      Merge pull request #3863 from getsentry/die-admin-die
      Merge pull request #3852 from getsentry/vendor-social-auth
      Remove manual connect of identities
      Vendor django-social-auth
      Initial support for Python 3
      Correct url name for linking
      Merge pull request #3853 from getsentry/python3
      Dont bind environment tag when used as attribute
      Restore full validation of empty values
      Force 2fa cookie to utf-8
      Correct import behavior to prevent Runtime error
      Fix handling of invalid IP networks
      Merge pull request #3871 from getsentry/fix-strict-networks
      Fix Python3 compat
      Bad ref to string_types
      Fix dist install requirements
      Python 3 compatibility
      Merge pull request #3872 from getsentry/python3
      Fix role downgrade permissions
      Merge pull request #3874 from getsentry/fix/role-downgrade
      Test proving managers cannot load api keys
      Correct scope of API key settings
      Merge pull request #3877 from getsentry/fix/api-key-security
      Return rendered label on rules
      Revert "Return rendered label on rules"
      Add label rendering for TaggedEventCondition
      Improve rendering of LevelCondition
      Correct/expand label tests
      Expand acceptance tests
      Correct handling of unicode file output
      Improve unicode handling in JS sources
      Merge pull request #3878 from getsentry/fix/unicode
      Quickly fail when no JSON data is found
      Merge pull request #3858 from getsentry/global-seen-stats
      Dont bind environment tag when used as attribute
      Merge pull request #3874 from getsentry/fix/role-downgrade
      Merge pull request #3877 from getsentry/fix/api-key-security
      Escape Sentry feedback embed config
      Merge pull request #3881 from getsentry/fix/escape-embed-config
      Remove references to sentry-stream (#3875)
      Remove unused docs code (#3850)
      Add get_context override on React pages
      Remove crumbs from shared events (#3880)
      Merge pull request #3876 from getsentry/improve-acceptance-tests
      Add server-side filters
      Various improvements to exec (#3894)
      Improvements to project creation endpoints
      Fix invalid cmp behavior for status checks (#3903)
      Bad reference to input
      Reduce rows-per-issue in weekly reports
      Merge pull request #3914 from getsentry/slim-issue-list
      Restrict Auth settings to Owner (#3918)
      Add user merge events to audit log (#3913)
      Merge pull request #3702 from getsentry/server-side-filters
      Merge pull request #3904 from getsentry/fix/team-project-perms
      Support foreignkey shortcuts for create_or_update
      Switch attribute to project
      handle fkey shortcut on values
      Merge pull request #3927 from getsentry/fix/create-update-fkey
      Fix propagation of project to tag receivers
      Only support column short for foreignkeys
      Utilize attname over type check
      Handle unauthenticated user (API)
      Add configuration for weekly reports (#3915)
      Add early adopter note to weekly reports
      Merge pull request #3935 from getsentry/early-adopter
      Enforce slug constraints in API (#3924)
      Improve handling of event type rendering
      Various hook additions/fixes for getsentry
      Add React compat layer for plugins
      Merge pull request #3949 from getsentry/plugin-react-compat
      Allow identity removal via API
      Allow rich data in contexts
      Utilize six.raise_from on lock exceptions (#3952)
      Improve membership invite flow
      Promote new discussion forum
      Update GitHub guidelines
      Enforce max length on project name (#3982)
      Remove stacktrace grouping logic for inlined JS (#3937)
      Sort releases by creation date when available
      Capture SDK client IP
      Remove OrganizationMember.counter usage and references
      Merge pull request #3970 from getsentry/feature/api-delete-identity
      Merge pull request #3971 from getsentry/feature/rich-contexts
      Merge pull request #3972 from getsentry/ref/remove-member-counter
      Merge pull request #3942 from getsentry/feature/sort-releases-by-date
      Merge pull request #3950 from getsentry/capture-client-ip
      Merge pull request #3938 from getsentry/fix/event-titles
      Merge pull request #3981 from getsentry/feature/improve-invites
      Correct tests
      Bad ref to legacy_token
      Correct other failing tests
      Ensure CSRF cookie name is configurable (#3988)
      Add server-side data filters (#3921)
      Rename Blacklist to Filter*
      Add tools for listing and removing authenticators (#4042)
      Overhaul plugin API for future React support (#3986)
      Fix rate limit editor + tests (#4047)

James Cunningham (7):
      Event-ize various natural language log events. (#3969)
      Change from using the event_id hex to the id in postgres. (#3978)
      Make the merge event string static. (#3979)
      Rehaul deletion logging. (#4014)
      Patch usage of period in key. (#4020)
      Expand on feedback from more verbose deletion logging. (#4022)
      Missed the other bulk delete log line. (#4023)

Jess MacQueen (24):
      wip plugins to react
      add tests
      pr feedback
      use group.stale instead of a special plugin group action
      js pr feedback
      pylint
      remove unused variable and set a default error message
      Merge pull request #3688 from getsentry/plugin-react
      fix Cannot read property 'forEach' of undefined
      Merge pull request #3898 from getsentry/fix-plugin-typeerror
      fix a few ui issues with react plugins
      Merge pull request #3899 from getsentry/plugin-ui-tweaks
      add status kwarg to sentry.plugins.base.response.JSONResponse (#3889)
      fix textarea cursor jumping to last character
      Merge pull request #3954 from getsentry/fix-textarea
      don't blow up if crumb.data is missing or null
      Merge pull request #3965 from getsentry/fix-JAVASCRIPT-19V
      fix Cannot read property 'substr' of undefined
      Merge pull request #3994 from getsentry/fix-JAVASCRIPT-1AG
      fix rest_framework.response.Response object is not JSON serializable
      fix Cannot read property 'isBookmarked' of undefined
      rewrite as one line
      Merge pull request #4013 from getsentry/fix-JAVASCRIPT-X2
      Merge pull request #3995 from getsentry/fix-response-not-serializable

Katie Lundsgaard (7):
      Fix tooltip width in event timestamp (#3887)
      Comment submit (#3888)
      Prevent StrictClick event when anchor is clicked allowing external link to open (#3896)
      Allow up/down arrow key navigation of project search menu (#3944)
      Breadcrumbs filter (#3920)
      Fix links with hash to scroll to anchor when linked (#3964)
      Make exception frame not expandable when there is no surrounding context (#4026)

Matt Robenolt (52):
      Minify svg from GH-3866
      Remove my social-auth patch and just apply it now to our vendored version (#3869)
      Minify svg from GH-3866
      8.7.0
      8.8.0.dev0
      Don't hardcode app.getsentry.com as SSO url (#3857)
      Fix bin/dump-command-help
      Fix collectstatic after python3 compat changes
      Also escape apostrophes in html encoder (#3908)
      Add support for working around react-native's encoding comment (#3909)
      Buffer incr for GroupTagValue is overqualified hitting wrong index (#3912)
      Revert "Buffer incr for GroupTagValue is overqualified hitting wrong index (#3912)"
      move project_id into the right place
      Prevent XSS through Request interface (#3923)
      Add debug to reserved slugs
      Give 24 hours for org delete to actually happen (#3933)
      Exand reserved slugs and fix logic in slugify_instance (#3932)
      Stop redirecting back to "home" when logging out. (#3941)
      Improve logging for mailgun (#3958)
      Respond to mailgun with 200's (#3959)
      Use newest exception when deriving metadata (#3963)
      Look at the right POST params for mailgun (#3962)
      Allow {SUDO,CSRF} cookie settings to be overridden
      Stop trying to use system.url-prefix for url generation on the frontend
      Switch to docs.sentry.io (#3980)
      Return back a 404 on an attempt to access a insecure path (#3983)
      Let DRF actually handle an auth failure without erroring (#3985)
      Change csrf cookie name to `sc`
      Bump postgres version in Dockerfile.dev
      Tag values from CSP reports need to run through validation (#4001)
      Actually remove Organizationmember.counter column (#4003)
      Truncate User.email value to 75 characters (#4002)
      Block connectionstrenth.com and siteheart.net by default for CSP
      Allow newer requests to be installed
      Don't fire off a stampeded of Team/Project deletion tasks (#4006)
      Track a continuous transaction id record through deletion events (#4005)
      Convert all delay calls with countdown to apply_async (#4009)
      Make sure we json encode our error response for release webhook (#4011)
      Don't kick to org create screen for super users if their activeorg isn't found (#4012)
      Mark Group as PENDING_DELETION before queueing (#4019)
      Better guard against race condition when cleaning up dead riak connections (#4021)
      Ignore a NoneType error
      Also catch a ValueError that may happen
      Move /register/ -> /auth/register/ (#4028)
      Refer to sentry.io (#4048)
      Remove all of ReCAPTCHA (#4024)
      Don't explicitly say continuous=False when deleting teams and projects (#4052)
      Guard against junk data being sent over the network (#4051)
      Only log during bulk deletions if rows were deleted (#4053)
      Optionally leverage ujson for decoding initial store payloads (#4035)
      Make integrationdocs not utilize six (#4062)
      Don't blow up on duplicate project slug (#3833)

Mike Helmick (2):
      Fixes #3865, add support for safari pinned tabs (#3866)
      Fixes #3865, add support for safari pinned tabs (#3866)

ckj (2):
      Add spacing utility classes
      Override bootstrap variables rather than using 'important'

ted kaemming (19):
      Record checksum as part of event data if used for grouping. (#3868)
      Basic report implementation. (#3836)
      Fix date partitioning in report generation. (#3891)
      Make TSDB bounds behavior consistent across all methods. (#3860)
      Minor report updates. (#3906)
      More report updates. (#3926)
      Temporarily remove report unsubscribe link. (#3929)
      Support report unsubscribe options. (#3928)
      Add unique user frequency condition. (#3849)
      Add report backend for intermediate caching. (#3931)
      Add weekly reports to Celery schedule. (#3934)
      Fix report graph display when either point is 0. (#3936)
      Prevent scheduling report delivery to organization members without associated user accounts. (#3940)
      Add missing `DISTINCT` clause to personal report resolution count. (#3955)
      Improve accuracy of regression query for reports. (#3948)
      Add TSDB counter for Release instances. (#3974)
      Add top release list to reports. (#3984)
      Add issue type distribution chart to report emails. (#4027)
      Add labels to report series graph X axis. (#4031)
```

**Comment by tkaemming** [comment]
- Added the ability to use unique users affected by an issue within a time period as a condition when defining rules.
